src/vs/vscode.proposed.d.ts

```diff
@@ -2163,6 +2163,47 @@ declare module 'vscode' {
 	}
 	//#endregion
 
+	//#region FileSystemProvider stat readonly - https://github.com/microsoft/vscode/issues/73122
+
+
+	/**
+	 * The `FileStat`-type represents metadata about a file
+	 */
+	export interface FileStat {
+		/**
```

Comment:
Repeating all existing members of the interface shouldn't be required as TS merges them based on their name.